cwms-data-api/src/main/java/cwms/cda/data/dao/timeseriesprofile/TimeSeriesProfileDao.java

```diff
@@ -160,11 +162,12 @@ public TimeSeriesProfileList catalogTimeSeriesProfiles(String locationIdMask, St
                 profileList.get(0).getKeyParameter(), total))
             .pageSize(Math.min(timeSeriesProfileResults.size(), pageSize))
             .total(total)
-            .nextPage(encodeCursor(delimiter, String.format("%s",
+            .nextPage(profileList.size() >= pageSize && total > pageSize
```

Comment:
This is too much logic for a ternary

Reply:
Pulled logic out of builder and replaced ternary